android-core/src/main/java/com/mparticle/MParticle.java

```diff
@@ -1699,6 +1707,26 @@ public interface ResetListener {
     }
 
 
+    /**
+     * Rokt Integration
+     * */
+    public class Rokt{
+        protected Rokt(){
+
+        }
+        public void selectPlacements(String viewName,
+                            Map<String, String> attributes,
```

Comment:
there should be a second method here also with just 2 arguments, per https://chat.google.com/room/AAAACu7-w_Q/9srmdeyLAiA/TmzM07mEqMA?cls=10 in order the simplify the invocation of this method by the partner

Reply:
like brandon did here
https://github.com/mParticle/mparticle-apple-sdk/pull/339/files